package.json

```diff
@@ -95,7 +95,7 @@
     "postcss-rtl": "^1.3.3",
     "raw-loader": "^3.0.0",
     "sass-loader": "^7.0.1",
-    "terra-aggregate-translations": "^1.0.0",
+    "terra-aggregate-translations": "git+https://github.com/cerner/terra-aggregate-translations.git#supported-locales",
```

Comment:
Sibling PR: https://github.com/cerner/terra-aggregate-translations/pull/19

These changes can technically be merged without the changes in terra-aggregate-translations, but the intention would be to bump the minimum version to the version that releases with these changes.